tests/test_reconnect_logic.py

```diff
@@ -719,6 +720,8 @@ async def on_connect() -> None:
         protocol = cli._connection._frame_helper
         send_plaintext_hello(protocol)
         send_plaintext_connect_response(protocol, False)
+        await asyncio.sleep(0)
+        await asyncio.sleep(0)
```

Comment:
The connection runs in a task so it wasn't actually finished yet but because of enum bug it looked like it was